rosbag2_transport/test/rosbag2_transport/test_play_until.cpp

```diff
@@ -318,37 +220,44 @@ TEST_F(RosBag2PlayUntilTestFixture, play_should_return_false_when_interrupted)
 
 TEST_F(RosBag2PlayUntilTestFixture, play_until_overrides_playback_duration)
 {
-  InitPlayerWithPlaybackUntilAndPlay(
-    150 /* playback_until_timestamp_millis */, 1 /* num messages topic 1 */,
-    1 /* num messages topic 2 */, 50 /* playback_duration_millis */);
+  InitializePlayerWith()
+  .PlaybackUntilTimestampInMillis(150)
+  .PlaybackDurationInMillis(50)
+  .ExpectedNumberOfMessagesOnTopic1(1)
+  .ExpectedNumberOfMessagesOnTopic2(1)
+  .AndPlay();
 
   auto replayed_test_primitives = sub_->get_received_messages<test_msgs::msg::BasicTypes>(
     kTopic1_);
   EXPECT_THAT(replayed_test_primitives, SizeIs(Eq(1u)));
-  EVAL_REPLAYED_PRIMITIVES(replayed_test_primitives);
+  EvalReplayedPrimitives(replayed_test_primitives);
 
   auto replayed_test_arrays = sub_->get_received_messages<test_msgs::msg::Arrays>(
     kTopic2_);
   EXPECT_THAT(replayed_test_arrays, SizeIs(Eq(1u)));
-  EVAL_REPLAYED_BOOL_ARRAY_PRIMITIVES(replayed_test_arrays);
-  EVAL_REPLAYED_FLOAT_ARRAY_PRIMITIVES(replayed_test_arrays);
+  EvalReplayedBoolArrayPrimitives(replayed_test_arrays);
+  EvalReplayedFloatArrayPrimitives(replayed_test_arrays);
 }
 
 TEST_F(RosBag2PlayUntilTestFixture, playback_duration_overrides_play_until)
 {
-  InitPlayerWithPlaybackUntilAndPlay(
-    50 /* playback_until_timestamp_millis */, 2 /* num messages topic 1 */,
-    2 /* num messages topic 2 */, 150 /* playback_duration_millis */);
+  InitializePlayerWith()
+  .PlaybackUntilTimestampInMillis(50)
+  .PlaybackDurationInMillis(150)
+  .ExpectedNumberOfMessagesOnTopic1(2)
+  .ExpectedNumberOfMessagesOnTopic2(2)
+  .AndPlay();
 
-  auto replayed_test_primitives =
-    sub_->get_received_messages<test_msgs::msg::BasicTypes>(kTopic1_);
+  auto replayed_test_primitives = sub_->get_received_messages<test_msgs::msg::BasicTypes>(
+    kTopic1_);
```

Comment:
```suggestion
  auto replayed_test_primitives = 
    sub_->get_received_messages<test_msgs::msg::BasicTypes>(kTopic1_);
```
Please revert code formatting for better readability.